src/app/home/components/DataList/Plans/MigrationsTable.tsx

```diff
@@ -84,7 +94,7 @@ const MigrationsTable: React.FunctionComponent<IProps> = ({ migrations, isPlanLo
           TODO: ** need to evaluate what text to show here **
           </Title> */}
         </EmptyState>
-      </Bullseye>
+      </Bullseye >
```

Comment:
Wondering why this isn't failing a lint rule? There shouldn't be whitespace here.